### ISSUE TITLE
Open Discussion

### ISSUE THREAD

**Issue by GeorgeS2019** [barcoderdev/unitypackage_godot#13]
Great Job!
Great **contribution** to Godot Community!
Consider open the Github Discussion

**Comment by ee0pdt** [comment]
Agree. Absolutely love this project! Thank you so much for making it!

**Comment by GeorgeS2019** [comment]
We all WANT godot **GREAT**!
[[WIP] Porting Unity C# to Godot C#](https://github.com/V-Sekai/unidot_importer/discussions/17)
